src/main/java/bio/terra/service/resourcemanagement/google/GoogleBucketService.java

```diff
@@ -280,12 +280,16 @@ private Bucket newCloudBucket(GoogleBucketResource bucketResource) {
     GoogleProjectResource projectResource = bucketResource.getProjectResource();
     String googleProjectId = projectResource.getGoogleProjectId();
     GcsProject gcsProject = gcsProjectFactory.get(googleProjectId);
+    GoogleRegion region =
+        bucketResource.getRegion() == GoogleRegion.US
+            ? GoogleRegion.US.getRegionOrFallbackBucketRegion()
+            : bucketResource.getRegion();
```

Comment:
Why can't we just use region.getRegionOrFallbackBucketRegion()?

Reply:
Ya know, that's exactly why we implemented that...